### PR TITLE
Make pre-release visual tweaks and improvements.

### PR DESCRIPTION
**Pull request by MichaelRoytman**
### Description

This pull request contains various UI improvements.

These improvements include:

- replacing the call-to-action in the user disclosure to a form input
- consolidating the form input between the user disclosure and the sidebar into one component
- modifying the styling of the form input
- replacing native HTML elements with Paragon components in the form input
- replacing the toggle button with a logo button
- small UI tweaks like changes to font size

JIRA: None.

### Demo

https://github.com/edx/frontend-lib-learning-assistant/assets/11871801/36128dbc-9879-421b-a288-98def14d716b

### CODE REVIEW

File: src/components/Disclosure/Disclosure.scss [hunk 1]
```diff
@@ -8,6 +8,10 @@
 
     h2 {
         font-size: 2rem;
+
+        .beta-subscript {
```
Comment (MichaelRoytman) [inline src/components/Disclosure/Disclosure.scss:12]: NOTE: I need to update this to `alpha-subscript` or `version-subscript`.

File: src/components/Disclosure/Disclosure.scss [hunk 2]
```diff
@@ -28,4 +32,8 @@
     li {
         font-size: 1rem;
     }
+
+    .disclaimer {
+        font-size: 1rem;
```
Comment (MichaelRoytman) [inline src/components/Disclosure/Disclosure.scss:37]: The Privacy team requested that this text be larger. I'm waiting to hear back from Spencer on whether this is the right font size.

File: src/components/Disclosure/index.jsx [hunk 3]
```diff
@@ -1,67 +1,56 @@
...
-          privacy policy
-        </Hyperlink>
-        .
-      </p>
-      <Button
-        onClick={handleAcknowledgeDisclosure}
-        variant="inverse-primary"
-      >
-        Let&apos;s get started.
-      </Button>
```
Comment (MichaelRoytman) [inline src/components/Disclosure/index.jsx:60]: This button has been replaced by the `MessageForm` component.

File: src/components/Disclosure/index.jsx [hunk 4]
```diff
@@ -1,67 +1,56 @@
...
-          <ul>
-            <li>Could you explain how to multiply two numbers?</li>
-            <li>How should an essay be structured?</li>
-            <li>How does photosynthesis work?</li>
-          </ul>
+const Disclosure = ({ courseId }) => (
+  <div className="disclosure d-flex flex-column align-items-stretch px-4 py-3">
+    <h2 className="text-light-100">
+      Xpert
+      <sub className="beta-subscript">alpha</sub>
```
Comment (MichaelRoytman) [inline src/components/Disclosure/index.jsx:15]: This is new.

File: src/components/Sidebar/index.jsx [hunk 5]
```diff
@@ -120,26 +106,7 @@ const Sidebar = ({
...
-          className="mt-2 mb-1 mx-1 border-0"
-          type="submit"
-          data-testid="submit-button"
-          variant="inverse-primary"
-          disabled={apiIsLoading}
-          aria-label="submit"
-        >
-          <Send size="22" />
-        </Button>
-      </form>
```
Comment (MichaelRoytman) [inline src/components/Sidebar/index.jsx:142]: This has been replaced by the `MessageForm` component.

File: src/components/Disclosure/index.jsx [hunk 6]
```diff
@@ -1,67 +1,56 @@
...
-          </div>
-        </div>
-        <div className="text-light-100 font-weight-light pl-3 my-2">
-          <ul>
-            <li>Could you explain how to multiply two numbers?</li>
-            <li>How should an essay be structured?</li>
-            <li>How does photosynthesis work?</li>
-          </ul>
+const Disclosure = ({ courseId }) => (
+  <div className="disclosure d-flex flex-column align-items-stretch px-4 py-3">
```
Comment (ilee2u) [inline src/components/Disclosure/index.jsx:12]: Looks like we also need a test for this Disclosure interstitial, but everything else looks pretty good.

**Review by ilee2u — CHANGES_REQUESTED** [review]

**Review by spencertiberi — APPROVED** [review]

File: src/components/Disclosure/index.jsx (hunk 6 above)
Comment (MichaelRoytman) [reply src/components/Disclosure/index.jsx:12]: We have added tests for the disclosure in this pull request https://github.com/edx/frontend-lib-learning-assistant/pull/18. We've decided to review and merge it after release, so it's on hold right now.

**Review by ilee2u — APPROVED** [review]

**Pull request merged**